streams/src/test/java/org/apache/kafka/test/ProcessorTopologyTestDriver.java

```diff
@@ -453,18 +448,20 @@ public void close() {
     private MockConsumer<byte[], byte[]> createRestoreConsumer(final Map<String, String> storeToChangelogTopic) {
         final MockConsumer<byte[], byte[]> consumer = new MockConsumer<byte[], byte[]>(OffsetResetStrategy.LATEST) {
             @Override
-            public synchronized void seekToEnd(final Collection<TopicPartition> partitions) {}
+            public synchronized void seekToEnd(final Collection<TopicPartition> partitions) {
```

Comment:
original was better

Reply:
Sorry about these. I'll update the formatter config.